src/app/pages/dashboard/dashboard.page.ts

```diff
@@ -83,4 +83,7 @@ export class DashboardPage extends BasePageComponent {
 
   showDelegatedExaminerRekey = (): boolean =>
     this.appConfigProvider.getAppConfig().role === ExaminerRole.DLG;
+
+  testResultColSize = (): number =>
```

Comment:
have you checked this in landscape orientation?

Reply:
works in landscape